index.html

```diff
@@ -1,25 +1,338 @@
+          <i id="icon-video" class="fas fa-video style-icon"></i>
+          <i id="icon-grid" class="fas fa-th style-icon"></i>
+          <i id="icon-setting" class="fa fa-ellipsis-v style-icon"></i>
+          <button class="log-in">
+            <span> <i id="icon-profile" class="fa fa-user-circle"></i> </span
+            >SIGN IN
+          </button>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <div class="primary-column">
+        <span class="video">
+          <iframe
+            class="embed"
+            src="https://www.youtube.com/embed/ROOeGPrC1Do"
+            allowfullscreen
+          >
+          </iframe>
+        </span>
+
+        <div class="video-title">
+          <p id="title">
+            FULL FILM: McLaren Speedtail vs F35 Fighter Jet | Top Gear
+          </p>
+        </div>
+        <div>
+          <p id="views">
+            9,710,783 views • Apr 26, 2020
+            <span id="separate">
+              <span><i class="fas fa-thumbs-up"> 1.3k</i> </span>
+              <span> <i class="fas fa-thumbs-down"> 20</i></span>
+              <span> <i class="fas fa-share"> Share</i> </span>
+              <span> <i class="fas fa-cloud-download-alt"> Save</i> </span>
+              <span> <i class="fa fa-ellipsis-h"></i> </span>
+            </span>
+          </p>
+        </div>
+        <hr class="line" />
+        <div>
+          <img src="icons/ch.jpg" alt="channel image" id="ch-img" />
+          <p id="ch-name"><b> Top Gear</b></p>
+          <p id="ch-subscribers">7.33M subscribers</p>
+
+          <span><button id="subscribe">SUBSCRIBE</button></span>
+          <p id="content">
+            Hammond is in the dunes of Abu Dhabi to drive a new six wheeled
+            machine from Mercedes Benz – the G63 6x6. Originally developed for
+            the Australian military, it's now available civilians - as long as
+            you have over half a million dollars to spend!
+          </p>
+        </div>
+        <hr class="line" />
+
+        <div id="com-numbers">
+          <h2>
+            <span>14,239 Comments &emsp; &emsp;</span
+            ><span class="sort"
+              ><i class="fas fa-sort-amount-up"></i> SORT BY
+            </span>
+          </h2>
+          <br />
+          <div class="com-input">
+            <span
+              ><img
+                class="com-img"
+                alt="video"
+                src="https://pluspng.com/img-png/user-png-icon-male-user-icon-512.png"
+            /></span>
+            <input id="comment" placeholder="Add comment" />
+          </div>
+
+          <div class="com-block">
+            <div class="com-unity">
+              <span
+                ><img
+                  class="com-img"
+                  alt="video"
+                  src="https://lh3.googleusercontent.com/a-/AOh14Gg3z157SKsDNkciDW-e7qUfZIM01WXORpWQp8ShQA=s88"
+              /></span>
+
+              <div style="margin-left: 20px;">
```

Comment:
- Remove in-line CSS. It's a bad practice -> ([article](https://speckyboy.com/good-bad-css-practices)).